data_processors/pipeline/tools/liborca.py

```diff
@@ -237,3 +238,11 @@ def get_run_config_from_runinfo(gds_volume: str, runinfo_path: str) -> str:
     logger.info(f"Extracted run config: {run_config}")
 
     return json.dumps(run_config)
+
+def get_tiny_uuid() -> str:
```

Comment:
Could you please change to call it `generate_tiny_uuid()` instead? So that it reflects the fact that it will always return something "new".